lib/hdb/hdb.h

```diff
@@ -307,11 +307,9 @@ typedef struct HDB {
 #define HDB_INTERFACE_VERSION	11
 
 struct hdb_method {
-    int			version;
+    HEIM_PLUGIN_FTABLE_COMMON_ELEMENTS(krb5_context);
     unsigned int	is_file_based:1;
```

Comment:
Aha, this is the issue, yes, that I added a bit-field after `version`, while the plugin framework in `lib/base` assumes the first three fields are `version`, `init`, and `fini`.  Good catch!